tools/platyPSHelp/platyPSHelp.psm1

```diff
@@ -0,0 +1,582 @@
+<#
+.ExternalHelp help\platyPSHelp-help.xml
+#>
+function New-ServiceMarkdownHelp
+{
+    [CmdletBinding()]
+    Param(
+        [Parameter(ParameterSetName="ResourceManager", Mandatory=$True)]
+        [Parameter(ParameterSetName="ServiceManagement", Mandatory=$True)]
+        [String]$Service,
+
+        [Parameter(ParameterSetName="ResourceManager", Mandatory=$True)]
+        [Parameter(ParameterSetName="ServiceManagement", Mandatory=$True)]
+        [Parameter(ParameterSetName="Storage", Mandatory=$True)]
+        [ValidateSet("Debug", "Release")]
+        [String]$BuildTarget,
+
+        [Parameter(ParameterSetName="ResourceManager", Mandatory=$True)]
+        [Parameter(ParameterSetName="ServiceManagement", Mandatory=$True)]
+        [Parameter(ParameterSetName="Storage", Mandatory=$True)]
+        [String]$PathToRepo,
```

Comment:
Now that this is a module, you shouldn't need this info, so this can be optional.  To get the directory that contains your module, use:

```
$PSModule = $ExecutionContext.SessionState.Module
$PSModuleRoot = $PSModule.ModuleBase

```

Using this, we should be able to remove this as a required parameter